gmso/core/pairpotential_type.py

```diff
@@ -0,0 +1,117 @@
+import unyt as u
+import warnings
+
+from gmso.core.potential import Potential
+from gmso.utils.decorators import confirm_dict_existence
+from gmso.exceptions import GMSOError
+from gmso.utils._constants import PAIRPOTENTIAL_TYPE_DICT
+from gmso.utils.misc import unyt_to_hashable
+
+
+class PairPotentialType(Potential):
+    """A descripton of the nonbondedinteraction between 2 atomtypes.
+
+    This is a subclass of the gmso.core.Potential superclass.
+
+    PairPotentialType represents a pair potential type and includes the functional form describing
+    its nonbonded interactions that doesn't follow usual cross-average combining rules.
+    The functional form of the potential is stored as a
+    `sympy` expression and the parameters, with units, are stored explicitly.
+    The AtomTypes that are used to define the pair potential type are stored as
```

Comment:
Nit but the names of the atom types in the form of strings are being stored in `member_types`.